app/assets/stylesheets/govuk-component/_govspeak.scss

```diff
@@ -8,6 +8,12 @@
     text-align: start;
   }
 
+  &.rich-govspeak {
+    strong {
+      font-weight: bold;
```

Comment:
Probably [this line](https://github.com/alphagov/static/blob/master/app/assets/stylesheets/helpers/_reset.scss#L30).